core/src/main/resources/error/error-classes.json

```diff
@@ -195,7 +195,7 @@
   },
   "INVALID_ARRAY_INDEX_IN_ELEMENT_AT" : {
     "message" : [
-      "The index <indexValue> is out of bounds. The array has <arraySize> elements. To return NULL instead, use `try_element_at`. If necessary set <config> to \"false\" to bypass this error."
+      "The index <indexValue> is out of bounds. The array has <arraySize> elements. Use `try_element_at` to tolerate accessing element at non-existing index and return NULL instead. If necessary set <config> to \"false\" to bypass this error."
```

Comment:
`non-existing index` => `invalid index` ?

Reply:
Yes, that is better. Fixed.